src/isPalindrome.js

```diff
@@ -0,0 +1,9 @@
+export default function isPalindrome(input){
+  if (typeof input ==='string'){
+    const alphaNumeric = input.toLowerCase().replace(/[^\w]|_/g, "");
+    const reverse = alphaNumeric.split('').reverse().join('');
+    return reverse === alphaNumeric;
+  }else{
```

Comment:
Check spacing in this file, keep spacing consistent for readability. Specifically in your `if/else` statements.